src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -1144,6 +1144,11 @@ cover.fluid.voiding.title=Fluid Voiding Settings
 cover.fluid.voiding.advanced.title=Advanced Fluid Voiding Settings
 cover.item.voiding.title=Item Voiding Settings
 cover.item.voiding.advanced.title=Advanced Item Voiding Settings
+cover.voiding.label.disabled=Disabled
+cover.voiding.label.enabled=Enabled
+cover.voiding.tooltip=WARNING! Setting this to "Enabled" means that fluids or items WILL be voided.
+cover.voiding.disabled=Voiding Cover Disabled
```

Comment:
I would maybe even go for 
```suggestion
cover.voiding.message.disabled=Voiding Cover Disabled
```

Reply:
fixed in https://github.com/GregTechCEu/GregTech/pull/1414/commits/a76bc73309894100ae97ac3aa6fba4e02c1623dc